src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +24,25 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    private static final int MAX_LETTERS = 5;
+
+    public String getUniqueCharacters(String fileName) throws IOException {
+        Set<Character> letterSet = new TreeSet<>();
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName))) {
+            int charValue;
+            while ((charValue = reader.read()) != -1) {
+                letterSet.add(Character.toLowerCase((char) charValue));
+            }
+        }
+
+        letterSet.removeIf(symbol -> !Character.isLetter(symbol));
```

Comment:
let's just check if Character is letter on line 34 (before we add it to the set)